### ISSUE TITLE
Firefox-only Test failure, Empty vs non-empty comment button text (open issue)

### ISSUE THREAD

**Issue by miketaylr** [webcompat/webcompat.com#1809]
```
FAIL: firefox on any platform - Comments (auth) - Empty vs non-empty comment button text (open issue) (2621ms)

Error: [POST http://localhost:4444/wd/hub/session/3e9090ea-e6c1-42fd-8217-a159e73d5250/element/57/value / {"value":["test comment"]}] Expected [object Undefined] undefined to be a string

Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:48:19 -0700'

System info: host: 'testing-gce-125db0d8-4374-44c6-85a5-590dcae03084', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.0-93-generic', java.version: '1.8.0_144'

Driver info: org.openqa.selenium.firefox.FirefoxDriver

Capabilities [{moz:profile=/tmp/rust_mozprofile.oo8HdPA8FiIL, rotatable=false, timeouts={implicit=0, pageLoad=300000, script=30000}, pageLoadStrategy=normal, platform=ANY, specificationLevel=0, moz:accessibilityChecks=false, acceptInsecureCerts=false, browserVersion=55.0.2, platformVersion=4.4.0-93-generic, moz:processID=5524, browserName=firefox, platformName=linux}]

Session ID: 5c347ddf-c39a-445c-9751-77fc5c3a3da2

  at Test.registerSuite.Empty vs non-empty comment button text (open issue) [as test]  <tests/functional/comments-auth.js:49:12>

  at nextTickCallbackWith0Args  <node.js:489:9>

  at process._tickCallback  <node.js:418:13>
```

This doesn't fail in Chrome, looks like a Geckodriver bug: https://github.com/mozilla/geckodriver/issues/659

**Comment by miketaylr** [comment]
More than this test, if you look at https://github.com/webcompat/webcompat.com/pull/1788

**Comment by miketaylr** [comment]
This will be fixed by #1788.
